library/src/main/java/com/schibsted/spain/barista/BaristaScrollToAction.java

```diff
@@ -0,0 +1,46 @@
+package com.schibsted.spain.barista;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.action.ScrollToAction;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.v4.widget.NestedScrollView;
+import android.view.View;
+import android.widget.HorizontalScrollView;
+import android.widget.ScrollView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
+import static android.support.test.espresso.matcher.ViewMatchers.withEffectiveVisibility;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anyOf;
+
+public class BaristaScrollToAction implements ViewAction {
```

Comment:
What do you think about explaining what this ViewAction exists? I mean, if it's to fix something related with the NestedScroll, we could clarify it in the name or something like that...